### PR TITLE
zh-translation:/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md

### PR DESCRIPTION
**Pull request by scwang18**
ref: https://github.com/servicemesher/istio-official-translation/issues/367
    [ ] Configuration Infrastructure
    [x] Docs
    [ ] Installation
    [ ] Networking
    [ ] Performance and Scalability
    [ ] Policies and Telemetry
    [ ] Security
    [ ] Test and Release
    [ ] User Experience
    [ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+owner: istio/wg-networking-maintainers
+test: yes
+---
+此任务描述如何使用[Ingress Resource](https://kubernetes.io/docs/concepts/services-networking/ingress/)入口资源将Istio配置为在服务网格集群之外公开服务。
+
+{{< tip >}}
+建议使用[Istio Gateway](/docs/tasks/traffic-management/ingress/ingress-control/)而不是Ingress来利用Istio提供的完整功能集，例如丰富的流量管理和安全功能。
+{{< /tip >}}
+
+## 准备工作
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:15]: 请添加 id，如 `{#prepare}`

File: content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+请按照[Ingress网关任务](/docs/tasks/traffic-management/ingress/ingress-control/)中的[准备工作](/docs/tasks/traffic-management/ingress/ingress-control/#before-you-begin)、[确定ingress IP和ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)的说明进行操作。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:17]: 写中文的地址。

File: content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    $ curl -s -I -HHost:httpbin.example.com "http://$INGRESS_HOST:$INGRESS_PORT/headers"
+    HTTP/1.1 404 Not Found
+    ...
+    {{< /text >}}
+
+## 下一步
+
+### TLS
+
+`Ingress` 支持[TLS settings](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)设置。 Istio支持此功能，但是引用的 `Secret` 必须存在于`istio-ingressgateway` 部署的名称空间（通常是 `istio-system` ）中。 [cert-manager](/docs/ops/integrations/certmanager/)可用于生成这些证书。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:73]:
```suggestion
`Ingress` 支持[TLS](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)设置。 Istio支持此功能，但是引用的 `Secret` 必须存在于`istio-ingressgateway` 部署的名称空间（通常是 `istio-system` ）中。 [cert-manager](/zh/docs/ops/integrations/certmanager/)可用于生成这些证书。
```

**Review by kebe7jun — APPROVED** [review]

File: content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+`Ingress` 支持[TLS](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/#tls)设置。 Istio支持此功能，但是引用的 `Secret` 必须存在于`istio-ingressgateway` 部署的名称空间（通常是 `istio-system` ）中。 [cert-manager](/zh/docs/ops/integrations/certmanager/)可用于生成这些证书。
+
+### 指定路径类型{#specifying-path-type}
+
+Istio默认路径类型为精确匹配，除非路径以 `/*` 或 `.*` 结尾，在这种情况下，路径类型为前缀匹配。不支持其他正则表达式。
+
+在Kubernetes 1.18中，添加了一个新字段 `pathType` 。这允许将路径明确声明为`Exact` 或 `Prefix`。
+
+### 指定 IngressClass {#specifying-ingress-class}
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md:80]:
```suggestion
### 指定 `IngressClass` {#specifying-ingress-class}
```

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**